README.md

```diff
@@ -22,16 +22,38 @@ If you make changes to an existing chart, but do not change its version, nothing
 
 [Checklist to comply-with when doing the PR](./.github/PULL_REQUEST_TEMPLATE.md)
 
-- Title of the PR starts with chart name (e.g. [mychartname])
+- Title of the PR starts with type and scope, for more details check [Commit and PR tile guidelines](#commit-and-pr-tile-guidelines)
 - Chart Version bumped
 - Variables are documented in the README.md (or README.tpl in some charts)
-- Check GithubAction checks (like lint) to avoid merge-check stoppers 
+- Check GithubAction checks (like lint) to avoid merge-check stoppers
+- Changelogs and Release Notes are automated based on the commit messages using git-chglog
+
+#### Commit and PR tile guidelines
+In order to automatically generate a meaningful changelog PR titles must respect the following rules (the same title must be used when merging it)
+
+A Type must be specified, avalilable types are:
+- feat
+- new
+- fix
+- perf
+- refactor
+- chore
+- docs
+
+A Scope should be always present, a few examples:
+- (agent)
+- (sysdig-deploy)
+- (agent,node-analyzer,sysdig-deploy)
+
+Full PR title example
+`feat(agent,node-analyzer,sysdig-deploy): add automated changelogs`
```

Comment:
we've been asked to include, as part of the changelog, elaborated content like the list of the vulnerabilities addressed with a particular change. Example here: https://github.com/sysdiglabs/charts/pull/766/files#diff-a465eea3af5b1cf427e17ce8720fc5bb4e1136c508c2954675599a63961bde21R7
Could you please specify what would be the proper way to generate something like that in the Changelog?

Reply:
I've use git-chlog in the past and AFAIK it's possible to fetch the commit <body> for this kind of stuff.

git commit body is basically everything other than the first line, and could help here @ironashram

Reply:
indeed, thanks, i am going to look into how to add it properly

Reply:
fyi i added a way to add an extendend changelog entry and also a dedicated section with an example in the `README`, it is not perfect since one will need to remember to add this section while squashing and merging, but i haven't found a better way